src/main/java/org/broadinstitute/hellbender/tools/walkers/rnaseq/ASEReadCounter.java

```diff
@@ -5,6 +5,7 @@
 import htsjdk.samtools.SAMFileHeader;
 import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
 import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.programgroups.RNAProgramGroup;
```

Comment:
This tool is now part of the CoverageAnalysisProgramGroup. See https://github.com/broadinstitute/gatk/pull/3924.

Reply:
Since the PR instituting this new programgroup just got merged, and I have started updates on the branch before this merge, I will do a series of commits to fix the program grouping.